editor/test/integration/gui_clipping_test.clj

```diff
@@ -13,16 +13,11 @@
 ;; specific language governing permissions and limitations under the License.
 
 (ns integration.gui-clipping-test
-  (:require [clojure.test :refer :all]
-            [clojure.data :as data]
+  (:require [clojure.data :as data]
+            [clojure.test :refer :all]
             [dynamo.graph :as g]
-            [integration.test-util :as test-util]
-            [editor.workspace :as workspace]
-            [editor.defold-project :as project]
             [editor.gui :as gui]
-            [editor.gl.pass :as pass]
-            [editor.handler :as handler]
-            [editor.types :as types]))
+            [integration.test-util :as test-util]))
```

Comment:
The changes to the editor tests should mirror the changes to the runtime tests in `test_gui_clipping.cpp`. However, the editor has a few more tests than the runtime for rendering, so there may be additional changes.